docs/Analyzer Configuration.md

```diff
@@ -792,3 +792,15 @@ Option Values: `true` or `false`
 Default Value: `false`
 
 Example: `dotnet_code_quality.CA1826.exclude_ordefault_methods = true`
```

Comment:
Unrelated, this is undocumented in dotnet/docs. We should review user options and make sure they're documented.

Reply:
@Youssef1313 Did you already file an issue for this?